src/seedu/addressbook/data/person/Unit.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.person;
+
+public class Unit {
+	
+	private boolean isPrivate;
+	private String unitName;
+	
+	public Unit(){
+	}	
+	
+	public Unit(String unitName, boolean isPrivate) {
+        this.isPrivate = isPrivate;
+        this.unitName = unitName;
+    }
+	
+	@Override
+	public int hashCode() {
+        return unitName.hashCode();
+    }
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.